package/MDAnalysis/topology/LAMMPSParser.py

```diff
@@ -602,13 +605,31 @@ def _parse_box(self, header):
         return unitcell
 
 
+# Define the headers that define topology information from the lammps file
+# Any headers outside of this set will be ignored
+DUMP_HEADERS = {
+    "id": {"attr_class": Atomids, "default": None, "dtype": np.int32},
+    "mol": {"attr_class": [Resids, Resnums], "default": 1, "dtype": int},
+    "type": {"attr_class": Atomtypes, "default": 1, "dtype": object},
+    "mass": {"attr_class": Masses, "default": 1.0, "dtype": np.float64},
+    "element": {"attr_class": Elements, "default": None, "dtype": object},
+    "q": {"attr_class": Charges, "default": None, "dtype": np.float32},
+}
+
+
 class LammpsDumpParser(TopologyReaderBase):
-    """Parses Lammps ascii dump files in 'atom' format.
+    """Parses Lammps ascii dump files.
+    id, mol, type, mass, element and q columns are read
```

Comment:
Empty line between header and synopsis
```suggestion

    id, mol, type, mass, element and q columns are read
```

Reply:
I have added this change